cylc/flow/workflow_files.py

```diff
@@ -193,6 +193,12 @@ class WorkflowFiles:
     LOG_DIR = 'log'
     """Workflow log directory."""
 
+    LOG_INSTALL_DIR = os.path.join(LOG_DIR, 'install')
+    """Install log dir"""
+
+    LOG_VERSION_DIR = os.path.join(LOG_DIR, 'version')
+    """Version control log dir"""
+
```

Comment:
Would it make sense to move these to a subclass `Log` á la `Service` and `Install` below?